src/Home/Home.tsx

```diff
@@ -49,59 +73,281 @@ const Home = ({
     )
   }
 
-  const handleRequestPermission = () => {
+  const showBluetoothDisabledAlert = () => {
+    Alert.alert(
+      t("home.bluetooth.bluetooth_disabled_error_title"),
+      t("home.bluetooth.bluetooth_disabled_error_message"),
+      [
+        {
+          text: t("common.okay"),
+        },
+      ],
+    )
+  }
+
+  const { applicationName } = useApplicationInfo()
+  const appDownloadLink = "https://pathcheck.org"
+  // TODO: Replace with actual app download link
```

Comment:
I created this ticket on Trello https://trello.com/c/YVm6ejAI would you mind reviewing and then we could potentially remove the TODO?

Reply:
Perfect thank you for doing that!